packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -198,10 +198,22 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
           timestamp: new Date(),
         };
 
+        const preProcessedFilename = `${s3FileName}.from_converter`;
+
+        await s3Utils.s3
+          .upload({
+            Bucket: conversionResultBucketName,
+            Key: preProcessedFilename,
+            Body: JSON.stringify(conversionResult),
+            ContentType: "application/fhir+json",
+          })
+          .promise();
```

Comment:
can this throw an error in some weird edge cases? if so, I'd put this behind a try/catch so that we're not failing the whole thing if the debug file fails to write